packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -269,18 +273,27 @@ class MenuContent extends React.Component {
       </div>
     ) : <div />;
 
+    const defaultHeader = this.props.headerTitle.length > 0 ? (
+      <Arrange
+        align="center"
+        fill={<h1 className={cx(['header-title', 'main-header-title'])}>{this.props.headerTitle}</h1>}
+      />
+    ) : undefined;
+
     return (
       <Arrange
         className={cx('header')}
         fitEnd={closeButton}
-        fill={backButton}
+        fill={this.props.index > 0 ? subMenuHeader : defaultHeader}
```

Comment:
nit: can we update this be something like:
````jsx
let subMenuHeader = (
      <div
        role="button"
        onClick={this.props.onRequestBack}
        onKeyDown={this.onKeyDownBackButton}
        tabIndex="0"
        aria-label={backBtnText}
      >
        <Arrange
          align="center"
          fitStart={(
            <div className={cx('header-button')}>
              {backIcon}
            </div>
          )}
          fill={<h1 className={cx('header-title')}>{this.props.title}</h1>}
        />
      </div>
);

let header = <div />

if (this.props.index > 0) {
    header = subMenuHeader;
} else if (this.props.headerTitle.length > 0) {
    header = (
       <Arrange
          align="center"
          fill={<h1 className={cx(['header-title', 'main-header-title'])}>{this.props.headerTitle}</h1>}
         />
     );
}
...

eturn (
      <Arrange
        className={cx('header')}
        fitEnd={closeButton}
        fill={header}
```

Reply:
Updated here 423a343d8fa324eeab15d11a2cf2c87e4e36e6f7